websites/D/Discords.com/dist/metadata.json

```diff
@@ -8,7 +8,7 @@
     "category": "socials",
     "logo": "https://i.imgur.com/ZjtyAQx.png",
     "thumbnail": "https://i.imgur.com/LbR2uNX.png",
-    "version": "1.0.0",
+    "version": "1.0.1",
```

Comment:
Doesn't really matter but it probably will make more sense if it's `1.1.0`?

Reply:
Probably yeah. I'll change it.